src/main/java/core/basesyntax/service/DataConverterImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import core.basesyntax.domain.Fruit;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataConverterImpl implements DataConverterService {
```

Comment:
The class implementation should not begin with an empty line. Please remove the redundant empty line at the beginning of the class definition.